src/Diagnostics/atmos_les_default.jl

```diff
@@ -404,10 +489,21 @@ function atmos_les_default_collect(dgngrp::DiagnosticsGroup, currtime)
 
         # FIXME properly
         if isa(bl.moisture, EquilMoist)
-            if !iszero(thermo.moisture.q_liq)
+            if thermo.moisture.has_condensate
                 idx = (Nq * Nq * (eh - 1)) + (Nq * (j - 1)) + i
-                ql_gt_0_z[evk][idx] = one(FT)
-                ql_gt_0_full[idx] = one(FT)
+                qc_gt_0_z[evk][idx] = one(FT)
+                qc_gt_0_full[idx] = one(FT)
+
+                z = zvals[evk]
+                cld_top = max(cld_top, z)
+                cld_base = min(cld_base, z)
+            end
+        end
+        if isa(bl.moisture, NonEquilMoist)
+            if thermo.moisture.has_condensate
```

Comment:
My original thought was to call `has_condensate` given a thermo state (on the fly) here. But I now see that it's a bit unclear how to do that. I'm fine with this so long this isn't a mem/perf issue in terms of storing a `Bool` in the state along with all of the other `FT`s. cc @kpamnany

Reply:
I see a thumbs up. So I'm gonna assume it's fine :)